MANIFEST.in

```diff
@@ -0,0 +1,2 @@
+recursive-include src *.yaml
```

Comment:
looks like maybe you included this to make check-manifest pass?

I think it was telling you something more important: namely that all the `_auto_images` were included in your PR.  obviously, they shouldn't be (they come from the docs PR, where they are added to git ignore).  Lesson being: pay attention to check-manifest, use it to make sure that you're not committing things that you shouldn't be, etc...  Don't just do stuff to make all the tests pass

Reply:
also... if you _do_ have an issue with check-manifest... don't add a MANIFEST.in file (that's sort of an old approach that isn't really necessary in most cases).  Add the file to `tool.setuptools.package-data` in pyproject.toml

Reply:
in this case, let's just not use YAML.  The plate database is small enough and manageable enough that you should just include it as a `dict` in the `_well_plate_database`